tests/e2e/test_rosacli_config.go

```diff
@@ -0,0 +1,93 @@
+package e2e
+
+import (
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/openshift/rosa/tests/ci/labels"
+	"github.com/openshift/rosa/tests/utils/exec/rosacli"
+)
+
+var _ = Describe("rosacli config",
+	labels.Feature.Config,
+	func() {
+		defer GinkgoRecover()
+
+		var (
+			rosaClient     *rosacli.Client
+			originalConfig map[string]string
+			configName     []string
+			ocmService     rosacli.OCMResourceService
+		)
+
+		BeforeEach(func() {
+			By("Init the client")
+			rosaClient = rosacli.NewClient()
+			ocmService = rosaClient.OCMResource
+		})
+		AfterEach(func() {
+			By("Restore the original config")
+			for key, value := range originalConfig {
+				if key != "scopes" {
+					_, err := ocmService.SetConfig(key, value)
+					Expect(err).To(BeNil())
+				}
+			}
+		})
+
+		It("to set and get config via rosacli - [id:72166]", labels.High, labels.Runtime.OCMResources, func() {
+
+			configName = []string{
```

Comment:
This could be global to the file or ?

Reply:
configName is used for "Get config via rosacli" step, move it under the step.